.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+# dependabot.yml
+
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "eduardo-rodrigues"
+      - "andrzejnovak"
+    prefix: "chore(deps): "
```

Comment:
```suggestion
    prefix: "chore(deps): "
    ignore:
      - dependency-name: "actions/*"
        update-types: ["version-update:semver-minor", "version-update:semver-patch"]
```

Reply:
This is what we've set up in several packages, see for example https://github.com/scikit-hep/particle/blob/master/.github/dependabot.yml.